packages/core/src/components/button/_overrides.scss

```diff
@@ -213,21 +293,20 @@ $pt-gp-dark-icon-color: $g-white;
 // CUSTOM class for action buttons with new sizes
 .#{$ns}-button.#{$ns}-g-action-button {
   text-transform: uppercase;
-  padding-left: 16px;
-  padding-right: 16px;
+  padding-left: 24px;
+  padding-right: 24px;
 
-  &.#{$ns}-small,
-  &.#{$ns}-g-extra-small {
+  &.#{$ns}-g-extra-small,
+  &.#{$ns}-g-extra-extra-small {
```

Comment:
Can we export the classes in the Classes.ts file? 

In the Tabs file, for example, I added a comment adding some information about the available classes: 
![image](https://user-images.githubusercontent.com/36133677/175918246-bda2defb-3a60-450a-9693-734a4ef81eb4.png)


I'm worried that today we have a fresh memory of the button changes, but in the future, it could be quite challenging to remember all of the possibilities!

Reply:
I exported `-g-extra-small` classes and also the other created for the button (like `kind-g-primary` or `-g-action-button`, etc..) 😁 Thanks for the tip Luna!